config/schedule.yml

```diff
@@ -0,0 +1,4 @@
+fredmansky_job:
+    cron: "00 15 * 1-5,9-12 FRI"
```

Comment:
Do we want to be notified exactly at 15.00 or just before? We might want the heads up. Or be able to choose? Might be too advanced though..

Reply:
Good idea! I think we need to have a Fredmansky meeting to discuss this